src/games/strategy/engine/message/HasEndPointImplementor.java

```diff
@@ -0,0 +1,20 @@
+package games.strategy.engine.message;
```

Comment:
Yeah, this is not fun to review. When breaking out the inner classes, that should be the *only* thing in the PR. It is not clear what has changed.